modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/suspended/AsyncSyntax.kt

```diff
@@ -0,0 +1,41 @@
+package arrow.effects.typeclasses.suspended
+
+import arrow.Kind
+import arrow.core.Either
+import arrow.core.Left
+import arrow.core.Right
+import arrow.effects.typeclasses.Async
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlin.coroutines.startCoroutine
+
+interface AsyncSyntax<F> : MonadDeferSyntax<F>, Async<F> {
```

Comment:
I'd say we can join them if the idea is well received.

Reply:
We can definitely move them to each of their correspondent type classes in the next PR for the cases where `bind()` is in scope, otherwise direct syntax does not work but I believe all the candidates are already `Monad`

Reply:
Something to be determined would be if we want the suspended functions also exported as the rest of the type class. Putting them on the type class will increase the size considerably whereas now since in the suspended world there are no `Kind` we don't have to worry about `fix` and they are just aggregated in the continuation.